src/plugins/foxPage/foxPage.tsx

```diff
@@ -1,14 +1,95 @@
-import { RawText } from 'components/Text'
+import { SimpleGrid, TabList, Tabs } from '@chakra-ui/react'
+import { useTranslate } from 'react-polyglot'
+import { useHistory } from 'react-router'
+import { selectAssetById } from 'state/slices/selectors'
+import { useAppSelector } from 'state/store'
+
+import { FoxTab } from './components/FoxTab'
+import { Layout } from './components/Layout'
+import { Total } from './components/Total'
 
 export enum FoxPageTab {
   Fox = 'Fox',
   Foxy = 'Foxy',
-  OneFox = 'OneFox',
 }
 
-export const FoxPage = (props: any) => {
-  // TODO remove console log when implementing tabs
-  // eslint-disable-next-line no-console
-  console.log(props)
-  return <RawText>TODO Fox Page</RawText>
+export enum FoxPageRoutes {
+  Fox = '/fox/fox',
+  Foxy = '/fox/foxy',
+}
+
+const FoxAssetId = 'eip155:1/erc20:0xc770eefad204b5180df6a14ee197d99d808ee52d'
+const FoxyAssetId = 'eip155:1/erc20:0xdc49108ce5c57bc3408c3a5e95f3d864ec386ed3'
+const OneFoxAssetId = 'eip155:1/erc20:0x03352d267951e96c6f7235037c5dfd2ab1466232'
```

Comment:
oneFOX references should be removed

Reply:
I need it in order to fetch the related icon visible over `Total FOX value` in the mockup https://www.figma.com/proto/WgALgik9YuWzq7VIM0JRdb/FOX-Page-Bounty?node-id=71%3A533&scaling=min-zoom&page-id=71%3A532

Reply:
Since we're going to have only FOX and FOXy assets across the feature, Total FOX value shouldn't have oneFOX icon anymore, it's safe to remove it

Reply:
Removed it !